website/content/docs/secrets/kmip-profiles.mdx

```diff
@@ -0,0 +1,269 @@
+---
+layout: docs
+page_title: KMIP - Profiles Support
+description: |-
+  The KMIP profiles define the use of KMIP objects, attributes, operations, message elements
+  and authentication methods within specific contexts of KMIP server and client interaction.
+  These profiles define a set of normative constraints for employing KMIP within a particular
+  environment or context of use.
+---
+
+## [Baseline Server][baseline-server]
+1. <details>
```

Comment:
Do you think it is really necessary to use `<details>`? I think it would be fine for all the tables to be visible, since this document is devoted to KMIP conformance?

Reply:
Yeah, makes sense to remove details tag